rlua-builders-derive/src/lib.rs

```diff
@@ -115,7 +115,7 @@ fn enum_builder(name: Ident, de: DataEnum) -> TokenStream2 {
         .unzip();
 
     quote! {
-        impl<'s> LuaBuilder<'s, rlua::Table<'s>> for #name {
+        impl<'s> ::rlua_builders::LuaBuilder<'s, rlua::Table<'s>> for #name {
```

Comment:
Interesting, though will this make any difference? I think only if you're importing `rlua-builders-derive` directly. Still good to have.

Reply:
The way I broke it was by aliasing the derive macro like this:

```rust
use rlua_builders::LuaBuilder as RBLB;

#[derive(RBLB)]
struct Foo {
}
```

It would resolve the macro but then the LuaBuilder trait wouldn't resolve. I'm admittedly still learning myself, so I don't 100% understand the name resolution rules at play here, but this fixed it so :shrug: lol